lib/Valetudo.js

```diff
@@ -10,71 +10,6 @@ const EventEmitter = require("events");
 const SSHManager = require("./SSHManager");
 const Logger = require("./Logger");
 
-const Valetudo = function() {
```

Comment:
Could you leave out this file for now as well? This will create a bunch of merge conflicts, e.g. viomi refactored the readfile stuff.

Reply:
In my opinion you put too much miio specific code in Valetudo.js, without which you wouldn't have the problem.

Reply:
You’re right there’s a lot of stuff here that should go into MiioVacuum. Then this should only init MiioVacuum and setup the webserver & mqtt.
However, if I had continued the refactoring to include this file the problem with merge conflicts would remain.

Reply:
Anyway, I took the file out.